pkg/service/aws/cost.go

```diff
@@ -136,3 +144,145 @@ func (svc AWSController) GetWorkspacesCost(ctx context.Context, req *proto.GetWo
 
 	return costResponse, nil
 }
+
+func (svc AWSController) GetCostByTime(ctx context.Context, req *proto.GetCostByTimeRequest) (*proto.GetCostByTimeResponse, error) {
```

Comment:
this is big function, makes hard to read, can you add some contextual comments on what the following logic does ?

Reply:
done